src/global.h

```diff
@@ -234,8 +234,8 @@ LED bar:      lbr
 // common tool tip bottom line text
 #define TOOLTIP_COM_END_TEXT             \
     "<br><div align=right><font size=-1><i>" + \
-    QCoreApplication::translate ( "global", "For more information use the ""What's " \
-    "This"" help (help menu, right mouse button or Shift+F1)" ) + \
+    QCoreApplication::translate ( "global", "For more information use the \"What's " \
```

Comment:
Do translations follow automatically via a different process or do they also need attention in this commit?

Reply:
Good call. I’m not yet familiar with the translation system, so will check.

Reply:
Translation files updated to match and included in this PR. Tested on Linux.